packages/perseus/src/interactive2/wrapped-defaults.ts

```diff
@@ -84,4 +87,92 @@ const WrappedDefaults: any = _.extend(
     }),
 );
 
+abstract class WrappedDefaults {
+    graphie: Graphie;
+    abstract wrapper: HTMLDivElement;
+    abstract visibleShape: {
+        /*
+         * These functions, when called on the wrapped object, simply pass the
+         * arguments to the underlying Raphael object.
+         */
+        attr: (...args: any[]) => void;
+        animate: (...args: any[]) => void;
```

Comment:
It's probably worth creating more specific types for `attr` and `animate`. Maybe we should also pull out an interface for the type of VisibleShape (or maybe one exists already?)